contracts/actions/aaveV3/AaveV3Borrow.sol

```diff
@@ -31,14 +31,25 @@ contract AaveV3Borrow is ActionBase, AaveV3Helper {
 
         params.amount = _parseParamUint(params.amount, _paramMapping[0], _subData, _returnValues);
         params.to = _parseParamAddr(params.to, _paramMapping[1], _subData, _returnValues);
-        params.market = _parseParamAddr(params.market, _paramMapping[2], _subData, _returnValues);
+        params.rateMode = uint8(_parseParamUint(uint8(params.rateMode), _paramMapping[2], _subData, _returnValues));
+        params.assetId = uint16(_parseParamUint(uint16(params.assetId), _paramMapping[3], _subData, _returnValues));
+        params.useDefaultMarket = _parseParamUint(params.useDefaultMarket ? 1 : 0, _paramMapping[4], _subData, _returnValues) == 1;
+        params.useOnBehalf = _parseParamUint(params.useOnBehalf ? 1 : 0, _paramMapping[5], _subData, _returnValues) == 1;
+        params.market = _parseParamAddr(params.market, _paramMapping[6], _subData, _returnValues);
         params.onBehalf = _parseParamAddr(
             params.onBehalf,
-            _paramMapping[3],
+            _paramMapping[7],
             _subData,
             _returnValues
         );
 
+        if (params.useDefaultMarket) {
```

Comment:
by replacing this from parseParams function here, we make executeActionDirect function not work the same way as the executeActionDirectL2 and executeAction. Currently not an issue, but if Aave V3 were to be deployed to mainnet, the sdk would call executeActionDirect and the function would not behave as we want it to. Maybe executeActionDirect should be empty function while this contract is only for L2 usage

Reply:
Same thing to be looked in other action contracts